src/Razor/src/Microsoft.VisualStudio.Editor.Razor/TelemetryReporter.cs

```diff
@@ -247,6 +254,8 @@ public IDisposable TrackLspRequest(string lspMethodName, string languageServerNa
         }));
     }
 
+    private static bool IsNumeric(object? value) => double.TryParse(Convert.ToString(value), out _);
```

Comment:
The `ellapsedms` looks like its being added as a `long`. Can we just change this to `value is long`? or `value is long or double or int` if we want to be a little more future proof?